src/main/java/com/bobocode/blyznytsia/bibernate/util/SqlUtil.java

```diff
@@ -0,0 +1,80 @@
+package com.bobocode.blyznytsia.bibernate.util;
+
+import static com.bobocode.blyznytsia.bibernate.util.EntityUtil.getEntityNonIdFields;
+import static com.bobocode.blyznytsia.bibernate.util.EntityUtil.resolveEntityIdField;
+import static com.bobocode.blyznytsia.bibernate.util.EntityUtil.resolveEntityTableName;
+import static com.bobocode.blyznytsia.bibernate.util.EntityUtil.resolveFieldColumnName;
+import static java.util.stream.Collectors.joining;
+
+import java.lang.reflect.Field;
+import lombok.experimental.UtilityClass;
+
+/**
+ * Utility class for generating SQL statements
+ */
+@UtilityClass
+public class SqlUtil {
+
+  private static final String SELECT_STATEMENT_TEMPLATE = "SELECT * FROM %s WHERE %s = ?";
+  private static final String INSERT_STATEMENT_TEMPLATE = "INSERT INTO %s (%s) VALUES (%s)";
+  private static final String UPDATE_STATEMENT_TEMPLATE = "UPDATE %s SET %s WHERE %s = ?";
+  private static final String DELETE_STATEMENT_TEMPLATE = "DELETE FROM %s WHERE %s = ?";
+
+  /**
+   * Builds an SQL INSERT statement for the given entity type.
+   *
+   * @param entityType the entity type for which to build the statement
+   * @return the SQL INSERT statement
+   */
+  public static String buildInsertStatement(Class<?> entityType) {
+    var commaSeparatedColumns = getEntityNonIdFields(entityType).stream()
+        .map(EntityUtil::resolveFieldColumnName)
+        .collect(joining(", "));
```

Comment:
`", "` could it be defined as a constant as it is used 3 time in this class

Reply:
fixed